src/core/server/saved_objects/migrations/core/migrate_raw_docs.ts

```diff
@@ -46,8 +45,7 @@ export class CorruptSavedObjectError extends Error {
 export async function migrateRawDocs(
   serializer: SavedObjectsSerializer,
   migrateDoc: MigrateAndConvertFn,
-  rawDocs: SavedObjectsRawDoc[],
-  log: SavedObjectsMigrationLogger
```

Comment:
unused argument

Reply:
FWIW, This is where the [major change](https://github.com/elastic/kibana/pull/96986/files#diff-d900f155ce96b3a353753d1ecdcee0537c42d8e45d238cfd1d3d6dc34bee49dfR93) for handling https://github.com/elastic/kibana/issues/90279 is. There are other changes to the model too, where we're splitting up `OUTDATED_DOCUMENTS_TRANSFORM` step into 2 steps: the first one handles the actual document transforms then hands those off to the [next step](https://github.com/elastic/kibana/pull/96986/files#diff-9ee8edf8b1c5b2e5fd0b59597fafcbace91cd5164d8f9e2e2ae84b1d7704c0a7R642): `TRANSFORMED_CODUMENTS_BULK_INDEX` if there [weren't any failures](https://github.com/elastic/kibana/pull/96986/files#diff-9ee8edf8b1c5b2e5fd0b59597fafcbace91cd5164d8f9e2e2ae84b1d7704c0a7R639).

Reply:
Do you want me to revert this change?

Reply:
Not necessarily, I can rework it too as long as I don't break anything you've implemented 😉